lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -602,7 +603,11 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
                 location, deployment_parameters = self._create_deployment_parameters(
                     resource_group_name, environment, log
                 )
-
+                deployment_parameters_message = DeploymentMessage()
```

Comment:
Can it reuse the environment message?

Reply:
we can get the value from environment.runbook.nodes_requirement[0].features.items, can I pass environment.runbook as the message?